buildshiprun/handler.go

```diff
@@ -422,7 +422,19 @@ func enableStatusReporting() bool {
 	return os.Getenv("report_status") == "true"
 }
 
-func reportStatus(status *sdk.Status) {
+func reportStatus(status *sdk.Status, SCM string) {
+	if SCM == "github" {
```

Comment:
I think that we should check if SCM is github or gitlab, and if not, we should return error about not supporting that SCM

Reply:
Done